server/utils/routes.js

```diff
@@ -1,6 +1,13 @@
+import basePathConfig from '../configs/basePathConfig'
+
+const basePath = basePathConfig || ''
```

Comment:
why do you need this? The other routes that existed don't use it.

Reply:
good catch [a13e560](https://github.com/AMP-SCZ/dpdash/pull/293/commits/a13e5607638067485e759753bc273f9ccf668355)